packages/vector_graphics_compiler/test/clipping_optimizer_test.dart

```diff
@@ -0,0 +1,115 @@
.891c.001-.115.011-.23.03-.343v-.014c.009-.058.027-.114.04-.168l1.84-6.971h6.776l-.007-.003z" fill="#EA4335"/><path d="M62.483 15.126l-.718 7.885a1.715 1.715 0 01-1.7 1.53 1.654 1.654 0 01-1.852-1.667 2.581 2.581 0 010-.343l-.086.53-.02.15a1.696 1.696 0 01-1.546 1.318 1.758 1.758 0 01-1.79-1.752 2.31 2.31 0 01.01-.343v-.013c.007-.058.017-.115.03-.172l1.486-7.096 3.126-.022 3.06-.005z" fill="#EA4335"/><path d="M62.483 15.148h3.275l-.248 7.893c-.01.057-.019.114-.035.17v.014c-.03.11-.071.216-.12.319a1.745 1.745 0 01-2.244.896 1.99 1.99 0 01-1.276-2.137l.648-7.155zm-16.156 0h-3.174l-2.138 6.837c-.014.055-.032.114-.042.167v.015c-.02.113-.03.228-.031.343a1.794 1.794 0 001.675 1.89 1.7 1.7 0 001.58-1.29h.008l2.122-7.962zm6.776 0h-3.419l-1.73 6.97a1.671 1.671 0 00-.042.169v.014a2.13 2.13 0 00-.03.343 1.792 1.792 0 001.673 1.89 1.704 1.704 0 001.582-1.29h.006l1.96-8.096z" fill="#81CA95"/><path d="M49.685 15.148h-3.358l-1.839 6.97c-.014.057-.032.115-.041.169v.014a2.264 2.264 0 00-.03.343 1.792 1.792 0 001.675 1.89 1.703 1.703 0 001.581-1.29h.007l2.005-8.096z" fill="#35A853"/><path d="M59.416 15.126l-3.126.021-1.485 7.096a1.716 1.716 0 00-.031.172v.014a2.308 2.308 0 00-.01.342 1.759 1.759 0 001.79 1.752 1.696 1.696 0 001.545-1.317l.021-.151 1.296-7.93z" fill="#81CA95"/><path d="M62.483 15.126h-3.067l-1.182 7.21a1.71 1.71 0 00-.025.171v.015a2.515 2.515 0 000 .343 1.654 1.654 0 001.853 1.666 1.714 1.714 0 001.7-1.529l.721-7.876zm-6.193.022h-3.187l-1.705 6.97c-.015.057-.032.115-.043.169v.014a1.97 1.97 0 00-.03.343A1.793 1.793 0 0053 24.534a1.617 1.617 0 001.58-1.29h.008l1.702-8.096z" fill="#35A853"/><path d="M40.94 26.533h68.963v-4.221H40.941v4.22z" fill="#35A853"/><path d="M54.314 42.22l3.623.005a1.558 1.558 0 001.557-1.552l.009-5.845a1.559 1.559 0 00-1.552-1.556h-3.622a1.561 1.561 0 00-1.558 1.552l-.009 5.845a1.55 1.55 0 001.552 1.55z" fill="#A8DAB5"/><path d="M46.155 53.943l-25.27.04-.04-26.081a2.001 2.001 0 011.993-1.998l21.28-.034a2.001 2.001 0 011.997 1.992l.04 26.08z" fill="#AFCBFA"/><path d="M22.838 25.904a2.006 2.006 0 00-1.91 1.416l-2.065 7.9a3.656 3.656 0 107.314-.01l1.199-9.313-4.538.007z" fill="#669DF7"/><path d="M46.032 27.28a2.008 2.008 0 00-1.918-1.41l-4.538.008 1.229 9.307a3.656 3.656 0 107.314-.011l-2.087-7.894zm-12.554-1.393l-6.102.01-1.199 9.312a3.66 3.66 0 003.664 3.651 3.66 3.66 0 003.65-3.663l-.013-9.31z" fill="#E8F0FD"/><path d="M33.478 25.887l.015 9.31a3.658 3.658 0 007.314-.012l-1.229-9.307-6.1.009z" fill="#669DF7"/><path d="M38.03 53.973l-9.02.013-.015-10.026a1.652 1.652 0 011.65-1.655l5.715-.009a1.654 1.654 0 011.656 1.65l.015 10.027z" fill="#E8F0FE"/><path d="M27.578 11.599h-.006l.052.124.025.061 4.552 10.938a.868.868 0 001.6-.008l4.412-10.873a5.75 5.75 0 00.496-2.47 5.763 5.763 0 00-11.526.143 5.74 5.74 0 00.395 2.085z" fill="#EA4335"/><path d="M32.946 10.998a1.787 1.787 0 100-3.575 1.787 1.787 0 000 3.575z" fill="#A50E0E"/></g><defs><clipPath id="clip0_2630_58398"><path fill="#fff" d="M0 0h64v64H0z"/></clipPath></defs></svg> ''';
```

Comment:
Consider formatting this in an editor as a multiline indendted string for readability - also consider moving to the end of the file or to the `test_svg_strings.dart` file.

Reply:
fixed.